hr_timesheet_sequence/models/__init__.py

```diff
@@ -0,0 +1,4 @@
+# © 2023 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+
+from . import account_analytic_line
```

Comment:
Although we usually keep the exactly module name as file name, with hr_timesheet we should make an exception (when functionality belongs to timesheets, and that's the case):
```suggestion
from . import hr_timesheet
```